tests/Page.php

```diff
@@ -0,0 +1,12 @@
+<?php
+declare(strict_types=1);
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Dev\TestOnly;
+
+/**
+ * Class Page
+ */
+class Page extends SiteTree implements TestOnly
```

Comment:
We should avoid adding duplicate files that will also exist in the root application.

Instead we should add `silverstripe/recipe-cms` to require dev, which will install those files to `app/code` for us :)